### DISCUSSION TITLE
How to set 'entry_icon'

### DISCUSSION THREAD

**Discussion by domferr** [mdgaziur/findex#51] [Q&A] [answered]
Hey, great job, findex works like a charm! Your screenshot has a search icon and I was setting my `settings.toml` to achieve the same. 
Unfortunately I don't know to what `entry_icon` should be set. Even if I set it to a path it doesn't work. Can you explain how to set it properly? Thank you!

**Comment by mdgaziur** [comment, marked as answer]
Hi there! You have to set icon name from the icon theme you are using.

**Reply by domferr** [reply]
Unfortunately, it doesn't work. I also tried with another icon name which I'm sure exists (I am using it on a GNOME extension I made). What can be the problem?

To exclude any problem related to CSS, I also tried with an empty style.css file. This is my settings.toml:
```
default_window_width = 600
min_content_height = 0
max_content_height = 400
name_match_highlight_color = "white"
decorate_window = false
close_window_on_losing_focus = true
query_placeholder = "Search for applications"
icon_size = 32
toggle_key = "<Shift>space"
min_score = 10
result_size = 10
entry_icon = "system-search-symbolic"
```

![Screenshot from 2024-05-31 11-11-02](https://github.com/mdgaziur/findex/assets/14203981/53e18704-6129-4244-8fbb-7fcda4b9527b)

System Information:
> findex 0.8.1
> Ubuntu 23.04, Wayland
> GNOME 44.3

**Reply by domferr** [reply]
Maybe the problem is that the latest release (0.8.1) doesn't have the `entry_icon` feature yet?

**Reply by mdgaziur** [reply]
> Maybe the problem is that the latest release (0.8.1) doesn't have the `entry_icon` feature yet?

Ahh, I myself was kinda confused. Yes. Currently you'll have to build from latest commit to get the feature :3 I'll make a release soon when I'm happy with the state.

**Reply by domferr** [reply]
That's great to hear! Can't wait for the next release :)
